tests-beta/android/MASVS-RESILIENCE/MASTG-TEST-0x39-1.md

```diff
@@ -0,0 +1,27 @@
+---
+title: Debuggable Flag Not Disabled in the AndroidManifest
+platform: android
+id: MASTG-TEST-0x39-1
+type: [static]
+weakness: MASWE-0067
+---
+
+## Overview
+
+This test case checks if the application has the debuggable flag set in the AndroidManifest.xml. If this flag is set, an attacker can attach a debugger, and e.g., read and modify internals of the application.
+
+!!! note Other ways to debug the application
+    Not enabling debugging in the AndroidManifest.xml does fully prevent all possibilities to debug the app. See @MASWE-0101 for more details on how to prevent debugging.
+
+## Steps
+
+1. View the AndroidManifest.xml, e.g., via @MASTG-TOOL-0121.
```

Comment:
TOOL-0121 is objdump (ios)

Reply:
we should use MASTG-TOOL-0123

Reply:
But even better, use a technique "Obtaining Information from the AndroidManifest" (or similar title) with 2 sections to 
- get the whole android manifest: can use apktool or jadx/jadx-gui
- or read properties from it: can use aapt

Reply:
Always better to use a tech. from a test rather than a tool.

Reply:
@TheDauntless I think this overlapped with landing another tool. Fixed.

@cpholguera I split it up into tech and tool.